repos/rust-lang/rust.toml

```diff
@@ -0,0 +1,53 @@
+org = "rust-lang"
+name = "rust"
+description = "Empowering everyone to build reliable and efficient software."
+bots = ["bors", "rustbot", "rfcbot"]
+
+[access.teams]
+bootstrap = "write"
+cargo = "write"
+clippy = "write"
+compiler = "write"
+compiler-contributors = "write"
+core = "admin"
+infra = "write"
+lang = "write"
+libs = "write"
+libs-api = "write"
+libs-contributors = "write"
+lang-ops = "write"
+mods = "write"
+opsem = "write"
+project-exploit-mitigations = "write"
+project-stable-mir = "write"
+release = "write"
+rustdoc = "write"
+rustdoc-contributors = "triage"
```

Comment:
```suggestion
```

team no longer exists (#1474). there now is `rustdoc-frontend` (different team). probably worth double checking other teams, too, dunno

Reply:
Yes, we should rerun the ground truth from GitHub, I will ask Mark to help me with it.